protected/commands/SequenceFixerCommand.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/**
+ * Class SequenceFixerCommand
+ * Run SQL queries to fix out-of-sync sequences
+ *
+ * @author Rija Menage <rija+git@cinecinetique.com>
+ * @license GPL-3.0
+ */
+class SequenceFixerCommand extends CConsoleCommand {
+
+    public function actionFixCurationLog($args) {
+        echo "Reset the sequence for curation_log".PHP_EOL;
+        $sql = "SELECT setval(pg_get_serial_sequence('curation_log', 'id'), coalesce(max(id),0) + 1, false) FROM curation_log";
+        Yii::app()->db->createCommand($sql)->execute();
+
+        return 0;
+    }
```

Comment:
Should we also include action to fix `id` sequence of table `file`?
```php
    public function actionFixFile($args) {
        echo "Reset the sequence for file".PHP_EOL;
        $sql = "SELECT setval(pg_get_serial_sequence('file', 'id'), coalesce(max(id),0) + 1, false) FROM file";
        Yii::app()->db->createCommand($sql)->execute();

        return 0;
    }
```

Reply:
Good thinking @kencho51. However the problem is not just for the ``curation_log`` and the ``file`` tables.

The issue you were having with attribute id may even originate from the same root cause.

I've come up with a more generic solution in the latest commit on this branch at rija/gigadb-webiste@8bacdbf1.